src/components/CategoryCard.tsx

```diff
@@ -0,0 +1,28 @@
+import { Category } from "@/services/types";
+import Link from "next/link";
+
+const CategoryCard = ({ category }: { category: Category }) => {
+  return (
+    <section className="text-center cursor-pointer my-2 py-2">
+      <div className="flex-col p-2">
```

Comment:
This is weird. Why a `section` with only a `div` inside? Can we just have one of them? Why the need for both?